src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -32,10 +33,18 @@ namespace Microsoft.PowerShell.EditorServices.Services
     /// </summary>
     internal class PowerShellContextService : IHostSupportsInteractiveSession
     {
-        private static readonly string s_commandsModulePath = Path.GetFullPath(
-            Path.Combine(
+        // This is a default that can be overriden at runtime by the user or tests.
+        private static string s_bundledModulePath = Path.GetFullPath(Path.Combine(
                 Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
```

Comment:
Wait @rjmholt is this going to be accurate?

Reply:
`GetExecutingAssembly` seems to be PowerShell itself, so this becomes `$psHome` (and isn't caught by tests because xUnit _forces_ us to use a distinct module path).

Reply:
How could this have even worked in the past?! I don't think it did.

Reply:
Hm, no, my VS Code updated and it got funky.